lib/DevTools.js

```diff
@@ -0,0 +1,57 @@
+import _ from 'underscore';
```

Comment:
Any reason this is JS and not TS?

Reply:
@AndrewGable I thought that, like the app, we were not allowed to write `.ts` files yet. But I can rewrite this to `.ts` if you want!

Reply:
@AndrewGable So should I rewrite it to `.ts` or do you want me to provide a `.d.ts` file?